include/ensmallen_bits/pso/pso.hpp

```diff
@@ -60,12 +60,14 @@ namespace ens {
  *
  *    double Evaluate(const arma::mat& x);
  *
+ * @tparam MatType Type of matrix to optimize.
  * @tparam VelocityUpdatePolicy Velocity update policy. By default LBest update
  *     policy (see ens::LBestUpdate) is used.
  * @tparam InitPolicy Particle initialization policy. By default DefaultInit
  *     policy (see ens::DefaultInit) is used.
  */
-template<typename VelocityUpdatePolicy = LBestUpdate,
+template<typename MatType = arma::mat,
```

Comment:
If we need to add the matrix type as a template parameter, we should add it last for reverse compatibility, and update `doc/function_types.md` accordingly for PSO.